docs/docs/integrations/ibm-watson-assistant.md

```diff
@@ -0,0 +1,114 @@
+
+<TLDR>
+
+The IBM Watson™ Assistant service combines machine learning, natural language understanding, and an integrated dialog editor to create conversation flows between your apps and your users.
+
+- From the [IBM Watson Assistant documentation](https://cloud.ibm.com/apidocs/assistant/assistant-v2#introduction)
+
+</TLDR>
+
+:::tip What you will learn
+
+- Finding your credentials on your IBM Cloud dashboard
+- How to connect your IBM Watson Assistant to Airy Core
+
+:::
+
+:::note Prerequisites
+
+- A running Airy Core installation (refer to the [Get
+  Started section](getting-started/installation/introduction.md))
+- An [IBM Cloud](https://www.ibm.com/cloud) account with a running [IBM Watson Assistant](https://cloud.ibm.com/catalog/services/watson-assistant) product
+
+:::
+
+## Step 1: Find your credentials on your IBM Cloud dashbaord
+
+Once that you have an [IBM Cloud](https://www.ibm.com/cloud) account with a running [IBM Watson Assistant](https://cloud.ibm.com/catalog/services/watson-assistant) (see prerequisites), you need to find your assistant's credentials. Connecting the [IBM Watson Assistant](https://cloud.ibm.com/catalog/services/watson-assistant) to Airy Core requires 3 configuration values: your IBM Watson Assistant instance's `URL`, `apiKey` and `Assistant ID`.
+
+- Find your [IBM Watson Assistant](https://cloud.ibm.com/catalog/services/watson-assistant) instance's `URL` and `apiKey` on the IBM Watson Assistant service page. To access this service page: select `ressource list` from your dashboard, select the `AI / Machine Learning` category, and then select your assistant. Note down these credentials.
+
+- Next, on your IBM Watson Assistant service page, click on the button `Launch Assistant`: this opens the assistant page. On your assistant page, click on the `settings` button on the left sidebar, at the bottom. This opens the settings page: note the `Assistant ID`.
+
+<SuccessBox>
+
+Congratulations! You are now ready to connect IBM Watson Assistant to your Airy Core instance 🎉
+
+</SuccessBox>
+<br />
+
+Connecting [IBM Watson Assistant](https://cloud.ibm.com/catalog/services/watson-assistant) to your Airy Core instance can be done through API request or through the [Airy Control Center UI](/ui/control-center/introduction).
+
+We cover both options in this section: keep reading for connecting [IBM Watson Assistant](https://cloud.ibm.com/catalog/services/watson-assistant) via
+API request, scroll down to the next section for connecting it via the [Airy Control Center UI](/ui/control-center/introduction).
+
+## Step 2 (option 1): Install and configure IBM Watson Assistant to your Airy instance via API request
+
+Send an API request to the [Components Install](/api/endpoints/components#install) endpoint to install the [IBM Watson Assistant](https://cloud.ibm.com/catalog/services/watson-assistant) on your Airy instance.
+
+The request body should be:
+
+```json
+{
+  "name": "airy-core/ibm-watson-assistant-connector"
```

Comment:
We don't need the repo now. in the installation payload

Reply:
ah true! will make the change 👍